Street Trees/STConfirmationPageViewController.swift

```diff
@@ -0,0 +1,168 @@
+//
+//  STConfirmationPageViewController.swift
+//  Street Trees
+//
+//  Created by Pedro Trujillo on 10/17/16.
+//  Copyright © 2016 Code for Orlando. All rights reserved.
+//
+
+import UIKit
+import StreetTreesTransportKit
+
+
+//**********************************************************************************************************************
+// MARK: - Protocol
+
+protocol STConfirmationPageViewControllerDelegate: NSObjectProtocol {
+    func ConfirmationFormViewController(form: STConfirmationPageViewController, didCompleteWithWufooForm anWufooForm: STTKWufooForm)
+}
+
+//**********************************************************************************************************************
+
+class STConfirmationPageViewController: UIViewController, UITableViewDataSource, UITableViewDelegate
+{
+    
+    //var aTree:STTKStreetTree?
+    var anAddress: STTKStreetAddress?
+    var aContact: STTKContact?
+    
+    //just initial test values for vars
+    var treeName = "Example Tree"
+    var treeImage : UIImage?
+    var address = "101 Example Ave"
+    var cityStateZip = "Orlando, Fl 32801"
+    var name = "Bob Bobington"
+    var phone = "407-555-5550"
+    var email = "bob@google.com"
+    
+    weak var  delegate:STConfirmationPageViewControllerDelegate?
+    
+
+    override func viewDidLoad()
```

Comment:
remove this function as the table view will should use a datasource to insert its content.